tests/test_sample_view.py

```diff
@@ -399,39 +399,39 @@ def test_request_multiple_sample_positions(self):
                     )
 
         # try when requesting sample positions more than we have in the lab
-        with (
-            self.assertRaises(ValueError),
-            self.request_sample_positions(
-                [{"prefix": "furnace_temp", "number": 5}], task_id
-            ),
-        ):
-            pass
+        with self.assertRaises(ValueError):  # noqa: SIM117
+            with (
+                self.request_sample_positions(
+                    [{"prefix": "furnace_temp", "number": 10000}], task_id
+                ),
+            ):
+                pass
 
     def test_request_multiple_sample_positions_multiple_tasks(self):
         task_id_1 = ObjectId()
         task_id_2 = ObjectId()
 
         with self.request_sample_positions(
-            [{"prefix": "furnace_temp", "number": 2}], task_id_1
+            [{"prefix": "furnace_temp", "number": 32}], task_id_1
         ) as sample_positions:
-            self.assertEqual(2, len(sample_positions["furnace_temp"]))
+            self.assertEqual(32, len(sample_positions["furnace_temp"]))
             self.assertTrue(
                 sample_positions["furnace_temp"][0].startswith("furnace_temp")
             )
             self.assertTrue(
                 sample_positions["furnace_temp"][1].startswith("furnace_temp")
             )
             with self.request_sample_positions(
-                [{"prefix": "furnace_temp", "number": 2}], task_id_2
+                [{"prefix": "furnace_temp", "number": 32}], task_id_2
             ) as sample_positions_:
-                self.assertEqual(2, len(sample_positions_["furnace_temp"]))
+                self.assertEqual(32, len(sample_positions_["furnace_temp"]))
                 self.assertTrue(
                     sample_positions_["furnace_temp"][0].startswith("furnace_temp")
                 )
                 self.assertTrue(
                     sample_positions_["furnace_temp"][1].startswith("furnace_temp")
                 )
             with self.request_sample_positions(
-                [{"prefix": "furnace_temp", "number": 4}], task_id_2
+                [{"prefix": "furnace_temp", "number": 33}], task_id_2
```

Comment:
can you explain why the test should return 33 instead of 4 and 32 instead of 2?

Reply:
Because `furnace_temp` now has 64 spots. In this test, we are testing if the request_sample_positions can return expected result if the request tries to book more sample positions than available. If you try to book more spots than the available spots, you will get None.

In this test, we have already booked 32 spots. So we tried to book 33 more -> it will not be able to get that.